Jellyfin.Server/Helpers/StartupHelpers.cs

```diff
@@ -57,6 +57,9 @@ public static void LogEnvironmentInfo(ILogger logger, IApplicationPaths appPaths
         logger.LogInformation("User Interactive: {IsUserInteractive}", Environment.UserInteractive);
         logger.LogInformation("Processor count: {ProcessorCount}", Environment.ProcessorCount);
         logger.LogInformation("Program data path: {ProgramDataPath}", appPaths.ProgramDataPath);
+        logger.LogInformation("Log Directory Path: {LogDirectoryPath}", appPaths.LogDirectoryPath);
+        logger.LogInformation("Config Directory Path: {ConfigurationDirectoryPath}", appPaths.ConfigurationDirectoryPath);
+        logger.LogInformation("Cache Path: {CachePath}", appPaths.CachePath);
```

Comment:
```suggestion
        logger.LogInformation("Log directory path: {LogDirectoryPath}", appPaths.LogDirectoryPath);
        logger.LogInformation("Config directory path: {ConfigurationDirectoryPath}", appPaths.ConfigurationDirectoryPath);
        logger.LogInformation("Cache path: {CachePath}", appPaths.CachePath);
```
Consistent with the other log lines